README.md

```diff
@@ -3,43 +3,54 @@
 [![License](https://img.shields.io/github/license/tablelandnetwork/basin-cli.svg)](./LICENSE)
 [![standard-readme compliant](https://img.shields.io/badge/standard--readme-OK-green.svg)](https://github.com/RichardLitt/standard-readme)
 
-> Continuously publish data from your database to the Tableland network.
+> Continuously publish data from your database or file uploads to the Tableland Vaults network.
 
-# Table of Contents
+## Table of Contents
 
-- [Install](#install)
-- [Postgres Setup](#postgres-setup)
-  - [Self-hosted](#self-hosted)
-  - [Amazon RDS](#amazon-rds)
+- [Background](#background)
+- [Usage](#usage)
+  - [Install](#install)
+  - [Postgres Setup](#postgres-setup)
   - [Supabase](#supabase)
-- [Create a vault](#create-a-vault)
-- [Start replicating a database](#start-replicating-a-database)
-- [Write a Parquet file](#write-a-parquet-file)
-- [Listing Vaults](#listing-vaults)
-- [Listing Events](#listing-events)
-- [Running](#running)
-- [Run tests](#run-tests)
-- [Retrieving](#retrieving)
+  - [Create a vault](#create-a-vault)
+  - [Start replicating a database](#start-replicating-a-database)
+  - [Write a Parquet file](#write-a-parquet-file)
+  - [Listing Vaults](#listing-vaults)
+  - [Listing Events](#listing-events)
+  - [Retrieving](#retrieving)
+- [Development](#development)
+  - [Running](#running)
+  - [Run tests](#run-tests)
+- [Contributing](#contributing)
+- [License](#license)
+
+## Background
+
+Textile Vaults is a secure and verifiable open data platform. The Vaults CLI is a tool that allows you to continuously replicate a table or view from your database to the network (currently, only PostgreSQL is supported). Or, you can directly upload files to the vault (currently, parquet is only supported)
 
-# Background
+> 🚧 Vaults is currently not in a production-ready state. Any data that is pushed to the network may be subject to deletion. 🚧
 
-Textile Vaults is a secure and verifiable open data platform. The Vaults CLI is a tool that allows you to continuously replicate a table or view from your database to the network. Currently, only PostgreSQL is supported.
+## Usage
 
-> 🚧 Vaults is currently not in a production-ready state. Any data that is pushed to the network may be subject to deletion. 🚧
+### Install
 
-# Usage
+You can either install the CLI from the remote source:
+
+```bash
+go install github.com/tablelandnetwork/basin-cli/cmd/basin@latest
```

Comment:
just noticed `...basin@latest`, it should be `github.com/tablelandnetwork/basin-cli/cmd/vaults@latest`